packages/browser-utils/src/metrics/browserMetrics.ts

```diff
@@ -450,6 +451,34 @@ export function _addMeasureSpans(
     attributes['sentry.browser.measure_start_time'] = measureStartTimestamp;
   }
 
+  // https://developer.mozilla.org/en-US/docs/Web/API/Performance/measure#detail
+  if (entry.detail) {
+    // Handle detail as an object
+    if (typeof entry.detail === 'object') {
```

Comment:
I would also ask to support arrays of primitives, here and as object props :pray: 

Listing some things in spans is very useful, and `SpanAttributeValue` supports this.

Reply:
We'll still handle this via serializing to JSON, so you'll get this in the UI. I'd rather not pay the bundle size cost of the extra array processing logic.